channel.go

```diff
@@ -370,7 +370,10 @@ func (ch *Channel) serve() {
 				if max := 1 * time.Second; acceptBackoff > max {
 					acceptBackoff = max
 				}
-				ch.log.Warnf("accept error: %v; retrying in %v", err, acceptBackoff)
+				ch.log.WithFields(
+					LogField{"error", err.Error()},
+					LogField{"backoff", acceptBackoff},
+				).Warn("Accept error, will wait and retry.")
```

Comment:
nit: intentional period at end of all log messages? I'm fine either way, we might want to make sure it's consistent within our code and with other projects